ethereum/core/src/main/java/org/hyperledger/besu/ethereum/vm/MessageFrame.java

```diff
@@ -871,6 +877,15 @@ public Boolean isPersistingState() {
     return isPersistingState;
   }
 
+  /**
+   * Returns whether Message calls will be persisted
+   *
+   * @return whether Message calls will be persisted
+   */
+  public Hash getTransactionHash() {
```

Comment:
This method is used to build the PrivateBlockMetadata, but I don't fully understand what the transaction hash is needed for ...

Reply:
this is the PMT transaction hash